carbon-design-system/carbon-addons-iot-react
# fix(DateTimePickerV2): support translated string for am/pm in TimePickerDropDown

Closes #3728 

**Summary**

- We need to pass in translated string for am/pm in 12 hours format

**Change List (commits, features, bugs, etc)**

- items_here

**Acceptance Test (how to verify the PR)**

- Go to single select [story](https://deploy-preview-3732--carbon-addons-iot-react.netlify.app/?path=/story/2-watson-iot-experimental-%E2%98%A2%EF%B8%8F-datetimepickerv2--single-select)
- Change time ask to hh:mm A
- See time changed with the amString, pmString defined in i18n object
- select date and time and see all fields showing am/pm strings correctly
- delete time manually and see time input work correctly
- click on clear button and open dropdown again
- see no errors on time input 
- focus on time input and show time values are populated with current time


**Regression Test (how to m

https://user-images.githubusercontent.com/24279794/221307510-71ac87d0-98e1-43c8-9415-5205a2e4a085.mov

Make sure this PR doesn't break old functionality)**





**Things to look for during review**
